addOns/openapi/src/main/resources/org/zaproxy/zap/extension/openapi/resources/Messages.properties

```diff
@@ -2,24 +2,30 @@
 
 openapi.api.action.importFile = Imports an OpenAPI definition from a local file.
 openapi.api.action.importFile.param.file = The file that contains the OpenAPI definition.
+openapi.api.action.importFile.param.target = The Target URL to override the server URL present in the definition.
 openapi.api.action.importUrl = Imports an OpenAPI definition from a URL.
 openapi.api.action.importUrl.param.url = The URL locating the OpenAPI definition.
-openapi.api.action.importUrl.param.hostOverride = The host to override the one present in the definition.
+openapi.api.action.importUrl.param.hostOverride = The Target URL (called hostOverride for historical reasons) to override the server URL present in the definition.
 
 openapi.cmdline.file.help = Imports an OpenAPI definition from the specified file name
 openapi.cmdline.url.help = Imports an OpenAPI definition from the specified URL
+openapi.cmdline.targeturl.help = The Target URL, to override the server URL present in the OpenAPI definition. Refer to the help for supported format.
 
 openapi.desc = Allows you to spider and import OpenAPI (Swagger) definitions 
 openapi.topmenu.import.importopenapi = Import an OpenAPI definition from the local file system
 openapi.topmenu.import.importopenapi.tooltip = The file must be a formal described OpenAPI definition.
 openapi.topmenu.import.importremoteopenapi = Import an OpenAPI definition from a URL
 openapi.topmenu.import.importremoteopenapi.tooltip = The contents must be a formal described OpenAPI definition.
 
-openapi.importfromdialog.labeltarget = Override Host:
+openapi.importfromdialog.labeltarget = Target URL:
 openapi.importfromdialog.importbutton = Import
 openapi.importfromdialog.pasteaction = Paste
 openapi.importfromdialog.invalidurl = Invalid URL:\n{0}
-openapi.importfromdialog.badoverride = Invalid Override Host: this should be of the form "host" or "host:port":\n{0}
+
+openapi.importfromfiledialog.badfile = Can't find the specified file
+openapi.importfromfiledialog.choosefilebutton = Choose File
+openapi.importfromfiledialog.labelfile = File Path:
+openapi.importfromfiledialog.title = Import OpenAPI Definition from file
```

Comment:
If this is really a dialog title could we make it `Import OpenAPI Definition from File`?

Reply:
Sure.

Reply:
Done.